client/js/views/Header.js

```diff
@@ -14,6 +14,10 @@ module.exports = Object.create( Object.assign( {}, require('./__proto__'), {
     onNavListClick(event) {
         this.emit('navigate', `/${ event.target.textContent.replace(/\s+/g, '') }`)
         var headers = [...event.target.parentElement.children].forEach(header => header.classList.remove('selected'))
+        
+        /* This doesn't seem to work.  For some reason I need the 'var headers'.  This will produce an error. */
```

Comment:
What is the error?

Try putting a semi colon on the previous line.

Also, what node version are you using ?